webpage/src/installation/fedora.md

```diff
@@ -54,22 +61,22 @@ dnf install qownnotes
 
 [Direct Download](https://download.opensuse.org/repositories/home:/pbek:/QOwnNotes/Fedora_40) (this example link is for Fedora 40)
 
-## QOwnNotes version updating notes for Fedora
+## QOwnNotes version-updating notes for Fedora
 
 ### Problems with GPG keys?
 
-Changes in Fedora's cryptographic policies can mean "old" repository keys are not automatically extended.
+Changes in Fedora's cryptographic policies can mean "old" (expired) repository keys are not _automatically_ extended.
```

Comment:
Can you please use emphasis, instead of underline?

Reply:
Isn't _ the same as * in Github's markdown?

Happy to change, however.